### DISCUSSION TITLE
ScriptReference/EditorApplication.Exit

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#4] [Notes]
# ScriptReference/EditorApplication.Exit

https://docs.unity3d.com/ScriptReference/EditorApplication.Exit.html

**Comment by vertxxyz** [comment]
If you want to exit the editor to simulate a crash, use the [`Utils.ForceCrash`](https://docs.unity3d.com/ScriptReference/Diagnostics.Utils.ForceCrash.html) method.
